server.go

```diff
@@ -1702,6 +1713,9 @@ func (s *Server) serveConn(c net.Conn) error {
 		if serverName != nil {
 			ctx.Response.Header.SetServerBytes(serverName)
 		}
+		if contentType != nil {
+			ctx.Response.Header.SetContentTypeBytes(contentType)
```

Comment:
Doesn't this always overwrite the header no matter if the user set it first or not?

Reply:
Well, isn't this called *before* the Handler? So it's just setting prior to user code.